### DISCUSSION TITLE
wind showcase isnt working in master?

### DISCUSSION THREAD

**Discussion by tamir-jether** [visgl/deck.gl#6112] [Q&A]
hey, i was trying to use the wind example:
https://github.com/visgl/deck.gl/tree/master/showcases/wind

and noticed it uses a really old version of deck.gl. 
trying to patch it to use the master version causes issues.

are there any plans on updating this example to fit the newer deck.gl versions?

**Comment by Pessimistress** [comment]
We do not have the developer resource to upgrade the wind example at the moment. If you wish to try I'm happy to discuss any problems that you encounter.

**Comment by ibgreen** [comment]
@zakjan I think you looked briefly at this?

**Reply by zakjan** [reply]
I remember seeing some errors too, but I'm sorry I don't remember details. I used a similar idea of WebGL 2 TransformFeedback in https://github.com/kamzek/deck.gl-particle, but the rest is different, because it's my pre-existing particle simulation, ported to deck.gl.
